### PR TITLE
Migrate to Go Modules

### PR DESCRIPTION
**Pull request by alex1989hu**
- Add Go Modules support to Dockerfile
  - move from GOPATH to let go mod works
- Update drone to use Go Modules
  - move from GOPATH to let go mod works
  - remove unnecessary golang dep

**NOTE:** This update keep kubernetes-1.12.3 version.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,11 @@
+module github.com/kontena/kubelet-rubber-stamp
+
+go 1.11
+
+require (
+	github.com/operator-framework/operator-sdk v0.4.0
+	k8s.io/api v0.0.0-20190918155943-95b840bb6a1f
+	k8s.io/apimachinery v0.0.0-20190913080033-27d36303b655
+	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
```
Comment (jnummelin) [inline go.mod:9]: Just curious, how did you decipher which "version" to take in to "match" the previously used versions?

**Review by jnummelin — APPROVED** [review]
LGTM, only one small question so I better understand how you came to these specific mod versions to use.

File: go.mod (hunk 1 above)
Comment (alex1989hu) [reply go.mod:9]: Frankly, I can't recall it. I played with `go build` / `go mod`. Additionally made several search, but my proposal is to update it to the latest which has more deterministic tagging - fortunately they [simplified version tagging](https://github.com/kubernetes/client-go/blob/master/INSTALL.md#add-client-go-as-a-dependency). I can make a PR on top of this PR to update dependencies if it is okay for you.

**Pull request merged**